### PR TITLE
feat(CX-3496): use recipientEmail in get in touch flow

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR solves [CX-3496]

### Description

If the user visits the get in touch flow via the "contact _NAME_" button, the specialist's name will be displayed in the title and the email will be sent to this specialist
Test here: https://swa-lp-redesign.artsy.net/

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3496]: https://artsyproduct.atlassian.net/browse/CX-3496?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx [hunk 1]
```diff
@@ -67,6 +69,13 @@ export const ConsignmentInquiry: React.FC<ConsignmentInquiryProps> = ({
   const handleRecaptcha = (action: RecaptchaAction) =>
     new Promise(resolve => recaptcha(action, resolve))
 
+  const recipientEmail = match.params.recipientEmail
+    ? match.params.recipientEmail
+    : null
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx:74]:
```suggestion
  const recipientEmail = match.params?.recipientEmail ?? null
```

File: src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx [hunk 2]
```diff
@@ -102,16 +120,34 @@ export const ConsignmentInquiry: React.FC<ConsignmentInquiryProps> = ({
...
 
+      console.log(
+        "[LOGD] input,  response, match, recipientEmail, !!recipientEmail= ",
+        input,
+        response,
+        match,
+        recipientEmail,
+        !!recipientEmail ? true : false
+      )
+
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx:131]: 👀

File: src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx [hunk 3]
```diff
@@ -102,16 +120,34 @@ export const ConsignmentInquiry: React.FC<ConsignmentInquiryProps> = ({
...
       const consignmentInquiryId =
         response.createConsignmentInquiry?.consignmentInquiryOrError
           ?.consignmentInquiry?.internalID
 
+      console.log("[LOGD] consignmentInquiryId = ", consignmentInquiryId)
+
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx:137]: 👀

File: src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx [hunk 4]
```diff
@@ -102,16 +120,34 @@ export const ConsignmentInquiry: React.FC<ConsignmentInquiryProps> = ({
...
-        router.push("/sell/inquiry/sent")
+
+        router.push(
+          !!recipientEmail
+            ? `/sell/inquiry/${recipientEmail}/sent`
+            : "/sell/inquiry/sent"
+        )
       }
     } catch (error) {
+      console.log("[LOGD] error = ", error)
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx:149]: 👀

File: src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx (hunk 2 above)
Comment (dariakoko) [reply src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx:131]: 👁️ 👁️

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx [hunk 5]
```diff
@@ -82,6 +84,10 @@ export const MeetTheSpecialists: React.FC = () => {
     })
   }
 
+  const clickContactSpecialist = email => {
+    router.push(`/sell/inquiry/${email}`)
+    trackContactTheSpecialistClick()
```
Comment (lordkiz) [inline src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx:89]: I think you want to trigger the tracking before pushing a screen onto the stack

File: src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx (hunk 4 above)
Comment (dariakoko) [reply src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx:149]: Omg i'm sorry for all these console logs
was fighting with the router and used them a lot

File: src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx (hunk 5 above)
Comment (dariakoko) [reply src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx]: i've replaced it with router link

File: data/schema.graphql [hunk 6]
```diff
@@ -2615,6 +2615,9 @@ type ArtworkVersion implements Node {
   # The Artwork Version attribution class
   attributionClass: AttributionClass
 
+  # Artwork condition description
+  condition_description: String
+
```
Comment (lordkiz) [inline data/schema.graphql:2620]: Why do we have these schema changes?

Comment (dariakoko) [reply data/schema.graphql:2620]: I just run `yarn sync-schema` and it pulled these changes too

**Pull request merged**